### PR TITLE
started work on drop down selection for search results

### PR DESCRIPTION
**Pull request by ccsmith13**
# Description

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Change Status

- [ ] Complete, tested, ready to review and merge
- [x] Complete, but not tested (may need new tests)
- [ ] Incomplete/work-in-progress, PR is for discussion/feedback

## How Has This Been Tested?

- [ ] Test A
- [ ] Test B

## Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] My code has been reviewed by at least one peer
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] There are no merge conflicts

### CODE REVIEW

**Comment by ccsmith13** [comment]
[Edamam API Request - Nutrition](https://trello.com/c/qNbeWQMJ/122-edamam-api-request-nutrition)

File: src/state/slices/slices.js [hunk 1]
```diff
@@ -93,7 +93,7 @@ export const Edamam = (search) => async (dispatch) => {
   dispatch(callItem());
   try {
     const response = await axios.get(
-      `${edamamAPI}/food-database/v2/parser?app_id=${edamamAppID}&app_key=${edamamAppKey}&ingr=${search}`
+      `https://api.edamam.com/api/food-database/v2/parser?app_id=8de772d5&app_key=ba31a7a9230043a9bc36135b1a432184&ingr=${search}`
```
Comment (dave-vazquez) [inline src/state/slices/slices.js]: be careful not to commit any api keys to the repo, apis can revoke credentials if their web-crawlers find keys published to repos

**Review by taty2010 — APPROVED** [review]
Thanks for figuring out the button issue and refactoring the code.

**Review by Afrodo1 — APPROVED** [review]

**Comment by ccsmith13** [comment]
[Dropdown Component for Unit Selection](https://trello.com/c/V1k8pX4R/130-dropdown-component-for-unit-selection)

**Pull request merged**